core/templates/pages/collection-editor-page/navbar/collection-editor-navbar-breadcrumb.component.ts

```diff
@@ -24,6 +24,10 @@ import { CollectionEditorPageConstants } from '../collection-editor-page.constan
 import { CollectionEditorRoutingService } from '../services/collection-editor-routing.service';
 import { CollectionEditorStateService } from '../services/collection-editor-state.service';
 
+interface TabNamesToHumanReadableNames {
+  [tabName: string]: string;
```

Comment:
Can `Record<string, string>` be used instead?

Reply:
Done